### PR TITLE
Finished translation - needs review for certain strings

### PR DESCRIPTION
**Pull request by GuilhermeStrice**
## Purpose
Merge latest changes of the portuguese translation

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[X] Other... Please describe: translation
```

### CODE REVIEW

**Comment by changeworld** [comment]
@GuilhermeStrice 
Thank you for your contribution.
There are some flaws in the fix.
Please check the following and make corrections.
- https://github.com/Covid-19Radar/Covid19Radar/blob/master/HOW_TO_TRANSLATE_CONTRIBUTE.md#notes-on-translation

File: Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf [hunk 1]
```diff
@@ -693,7 +693,7 @@
         </trans-unit>
         <trans-unit id="SubmitConsentPageDescription5" translate="yes" xml:space="preserve">
           <source>If you have been in contact with you within 14 days, you will know that you may have been contacted without any information that can identify you personally.</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Se você esteve em contato com você dentro de 14 dias, você saberá que você pode ter sido contatado sem qualquer informação que possa identificá-lo pessoalmente.</target>
+          <target state="needs-review-translation" state-qualifier="mt-suggestion">Se você esteve em contacto com você dentro de 14 dias, você saberá que você pode ter sido contatado sem qualquer informação que possa identificá-lo pessoalmente.</target>
```
Comment (changeworld) [inline Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf:696]: Replace `<target state="needs-review-translation" state-qualifier="mt-suggestion">` -> `<target state="translated">`

File: Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf [hunk 2]
```diff
@@ -777,12 +777,12 @@
         </trans-unit>
         <trans-unit id="InqueryMailBody" translate="yes" xml:space="preserve">
           <source>Name: \r\n Contact: \r\n Content of inquiry (Please select a category from the following): 1. How the app works, 2. App settings, 3. Use of the app (notification etc.) ), 4. Others \r\n Inquiry text: \r\n</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Nome: \r\n Contato: \r\n Conteúdo de consulta (Selecione uma categoria a seguir): 1. Como funciona o aplicativo, 2. Configurações do aplicativo, 3. Uso do aplicativo (notificação etc.) ), 4. Outros \r\n Texto de consulta: \r\n</target>
+          <target state="needs-review-translation" state-qualifier="mt-suggestion">Nome: \r\n Contacto: \r\n Conteúdo de consulta (Selecione uma categoria a seguir): 1. Como funciona o aplicativo, 2. Configurações do aplicativo, 3. Uso do aplicativo (notificação etc.) ), 4. Outros \r\n Texto de consulta: \r\n</target>
```
Comment (changeworld) [inline Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf:780]: Replace `<target state="needs-review-translation" state-qualifier="mt-suggestion">` -> `<target state="translated">`

**Review by changeworld — CHANGES_REQUESTED** [review]

File: Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf [hunk 3]
```diff
@@ -128,7 +128,7 @@
         </trans-unit>
         <trans-unit id="ButtonReset" translate="yes" xml:space="preserve">
           <source>Stop</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Restaurar</target>
+          <target state="translated" state-qualifier="mt-suggestion">Parar</target>
```
[the remark this answers is not in the document]
Comment (GuilhermeStrice) [reply Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf]: Fixed

**Review by GuilhermeStrice — COMMENTED** [review]
Fixed mt.suggestion

**Review by GuilhermeStrice — COMMENTED** [review]
Fixed

File: Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf (hunk 1 above)
Comment (GuilhermeStrice) [reply Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf:696]: That is not translated, how can you be in contact with yourself?

Comment (changeworld) [reply Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf:696]: GitHub is detecting changes. If you didn't intend to change it, don't change it.

**Review by changeworld — CHANGES_REQUESTED** [review]

File: Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf (hunk 1 above)
Comment (GuilhermeStrice) [reply Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf:696]: Search and replace thing, I'll revert that

**Review by GuilhermeStrice — COMMENTED** [review]
Revert

**Review by changeworld — APPROVED** [review]

**Comment by changeworld** [comment]
@GuilhermeStrice 
Thanks!

**Pull request merged**